pyServer/GuestFishWrapper.py

```diff
@@ -144,7 +152,7 @@ def RunCredentialScanner(self, targetDir, operationOutFile):
                 return
 
             # Run credential scanner
-            bash_command = ["mono", "--runtime=v4.0", "/CS_Latest/tools/CredentialScanner.exe", "-I", targetDir, "-S", "/CS_Latest/tools/Searchers/diskinspectsearchers.xml", "-O", output_file]
+            bash_command = ["mono", "--runtime=v4.0", "/CS_Latest/tools/CredentialScanner.exe", "-I", scanfile, "-S", "/CS_Latest/tools/Searchers/diskinspectsearchers.xml", "-O", output_file]
```

Comment:
Maybe add a comment here describing we're only running credscan against the content in the scanfile?

Reply:
added the comment